cucumber-jparest/src/test/java/uk/gov/homeoffice/digital/sas/cucumberjparest/testapi/TestApiRunner.java

```diff
@@ -3,17 +3,20 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
 import org.springframework.context.annotation.Profile;
 
 import io.swagger.v3.oas.models.OpenAPI;
 import io.swagger.v3.oas.models.info.Info;
 import io.swagger.v3.oas.models.info.License;
+import uk.gov.homeoffice.digital.sas.jparest.config.ApplicationConfig;
```

Comment:
Now that this is a bit more exposed, does it's name need reviewing?

Reply:
I am open to other name suggestions

Reply:
Ah, I took a quick look and nothing useful sprang to mind and it's not worth holding the PR up on